index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Twitter. It's what's happening.</title>
+    <link rel="stylesheet" href="./master.css">
+    <link href="/Users/KingBrutus/Desktop/Pursuit/Unit2/Pursuit-Core-Web-Twitter-CSS-Assignment/favicon.ico" rel="icon" type="image/x-icon" />
```

Comment:
the href for this should be `./favicon.ico` this is because here you are directing your file to go to the directory based on your computer. Since this directory doesn't exist on mine, it gives me an error.